build/configs/rtl8720e/rtl8720e_make_bin.sh

```diff
@@ -255,9 +230,6 @@ function copy_kr4_image()
 
 copy_bootloader;
 copy_flashloader;
+copy_km4_image3;
 copy_kr4_image;
-if [ "${CONFIG_BINARY_SIGNING}" == "y" ];then
-	concatenate_binary_with_signing;
```

Comment:
Could you let me know why you remove the signing?

Reply:
The image signing mechanism is not available in TP1e TizenRT master, we have provided a signing script.
All image signing and encryption can be done via script.

Reply:
In the public, signing and non-signing eventually calls the same (nothing). We keep this for easy maintenance. Could you keep it as it is?

Reply:
Hi @sunghan-chang , wanna double confirm that as you have mentioned that "signing and non-signing eventually calls the same (nothing)"
may I know is that doing as below way:

if [ "${CONFIG_BINARY_SIGNING}" == "y" ];then
	**concatenate_binary_without_signing;**
else
	**concatenate_binary_without_signing;**
fi

Thank You.

Reply:
@ZhenBei-Sin Could you refer rtl8721csm_make_bin.sh and rtl8721csm_signing.sh?

Reply:
Copied rtl8721csm_signing.sh be rtl8720e_signing.sh

if [ "${CONFIG_BINARY_SIGNING}" == "y" ];then
  concatenate_binary_with_signing;
else
  concatenate_binary_without_signing;
fi

concatenate_binary_with_signing will call to rtl8720e_signing.sh